File: Spring/Chapter7/MyBatisSpringJavaInit/src/main/java/pers/manyToMany/entity/OrdersWithProduct.java
package pers.manyToMany.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrdersWithProduct {
    private Integer id;
    private String ordersn;
    private List<Product> products;
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/main/kotlin/pers/manyToMany/entity/OrdersWithProduct.kt
package pers.manyToMany.entity

class OrdersWithProduct {
    var id = 0
    var ordersn = ""
    var products:List<Product> = mutableListOf()
    override fun toString() = "OrdersWithProduct(id:$id,ordersn:$ordersn,products:$products)"
}